nestjs/src/courses/team-distribution/team.service.ts

```diff
@@ -0,0 +1,96 @@
+import { Team } from '@entities/index';
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { paginate } from 'src/core/paginate';
+import { Repository } from 'typeorm';
+import { UpdateTeamDto } from './dto';
+
+@Injectable()
+export class TeamService {
+  constructor(
+    @InjectRepository(Team)
+    private repository: Repository<Team>,
+  ) {}
+
+  private generatePassword(length = 6): string {
+    const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+    let result = '';
+    for (let i = 0; i < length; i++) {
+      result += characters.charAt(Math.floor(Math.random() * characters.length));
+    }
+    return result;
```

Comment:
consider using `nanoid` - we already have it

Reply:
fixed